### PR TITLE
Gleam v0.30.5

### PR DESCRIPTION
**Pull request by aslilac**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Add a package for the Gleam programming language
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98430)

### CODE REVIEW

**Comment by aslilac** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @aslilac,

The package installs normally, but requires `Microsoft.VCRedist.2015+.x64` to run:
image

Pending:
* https://github.com/microsoft/winget-cli/issues/163

File: manifests/g/Gleam/Gleam/0.30.5/Gleam.Gleam.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+PackageIdentifier: Gleam.Gleam
+PackageVersion: 0.30.5
+InstallerType: zip
```
Comment (stephengillie) [inline manifests/g/Gleam/Gleam/0.30.5/Gleam.Gleam.installer.yaml:4]:
```suggestion
InstallerType: zip
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**